README.md

```diff
@@ -63,33 +31,18 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Vet_Clinic] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+This project is created to practice and test our knowledge in SQL and to learn best practices of it.
 
-**[your_project__name]** is a...
+**[Vet_Clinic]** is a SQL database project focused on practice and learn SQL basics.
```

Comment:
- [ ] @NelsonAraujo93 great work implementing the changes requested however you have added two descriptions of the project kindly remove one.